sw/device/lib/crypto/impl/kdf.c

```diff
@@ -28,8 +29,109 @@ otcrypto_status_t otcrypto_kdf_kmac(
     otcrypto_kmac_mode_t kmac_mode, const otcrypto_const_byte_buf_t kdf_label,
     const otcrypto_const_byte_buf_t kdf_context, size_t required_byte_len,
     otcrypto_blinded_key_t *keying_material) {
-  // TODO: Implement KMAC-KDF.
-  return OTCRYPTO_NOT_IMPLEMENTED;
+  // Check NULL pointers.
+  if (key_derivation_key.keyblob == NULL || keying_material == NULL) {
```

Comment:
nit:
```suggestion
  if (key_derivation_key.keyblob == NULL || keying_material == NULL || keying_material->keyblob == NULL) {
```
edit: I see now that you have the NULL-check for the keyblob later, but I still think it's more consistent with the rest of cryptolib if it's here.